app/views/spree/admin/orders/_invoice_table4.html.haml

```diff
@@ -0,0 +1,62 @@
+%table.order-summary{:width => "100%"}
+  %thead
+    %tr
+      %th{:align => "left"}
+        %h5= t(:invoice_column_item)
+      %th{:align => "right", :width => "15%"}
+        %h5= t(:invoice_column_qty)
+      %th{:align => "right", :width => "15%"}
+        %h5= @invoice_presenter.has_taxes_included ? t(:invoice_column_unit_price_with_taxes) : t(:invoice_column_unit_price_without_taxes)
+      %th{:align => "right", :width => "15%"}
+        %h5= @invoice_presenter.has_taxes_included ? t(:invoice_column_price_with_taxes) : t(:invoice_column_price_without_taxes)
+      - if @invoice_presenter.total_tax > 0
+        %th{:align => "right", :width => "15%"}
+          %h5= t(:invoice_column_tax_rate)
+  %tbody
+    - @order.sorted_line_items.each do |item|
+      %tr
+        %td
+          = render 'spree/shared/line_item_name', line_item: item
```

Comment:
I think you might need to use the `_invoice/line_item_name` partial here instead? 
```ruby
= render 'spree/admin/orders/_invoice/line_item_name', line_item: item
```